worker/utils.ts

```diff
@@ -1,16 +1,6 @@
 import * as fs from "fs";
 import * as path from "path";
-import { findPackageJson } from "@/server/versioning";
-
-export function getVersion(): string {
-  try {
-    const content = findPackageJson();
-    const packageJson = JSON.parse(content);
-    return packageJson.version || "unknown";
-  } catch (error) {
-    return "unknown";
-  }
-}
+import { getVersion } from "@/server/versioning";
```

Comment:
Bc we made changes in the worker directory, bump the version in the package json